client/core/core.go

```diff
@@ -237,15 +321,34 @@ out:
 				case msgjson.SuspensionRoute:
 					log.Info("suspension message received")
 				}
+
 			case msgjson.Notification:
 				switch msg.Route {
 				case msgjson.BookOrderRoute:
-					log.Info("book_order message received")
+					err := c.handleBookOrderMsg(dc, msg)
+					if err != nil {
+						log.Error(err)
+					}
+
 				case msgjson.EpochOrderRoute:
 					log.Info("epoch_order message received")
+
 				case msgjson.UnbookOrderRoute:
-					log.Info("unbook_order message received")
+					err := c.handleUnbookOrderMsg(dc, msg)
+					if err != nil {
+						log.Error(err)
+					}
 				}
+
+			case msgjson.Response:
+				switch msg.Route {
+				case msgjson.OrderBookRoute:
+					err := c.handleOrderBookMsg(dc, msg)
+					if err != nil {
+						log.Error(err)
+					}
+				}
```

Comment:
Even though this is all placeholder code, perhaps even because of that, it would be valuable to have `default`s logged as unknown routes so we know when we've forgotten to hook something up.